src/Umbraco.Headless.Client.Net/Configuration/IHeadlessConfiguration.cs

```diff
@@ -1,7 +1,11 @@
-﻿namespace Umbraco.Headless.Client.Net.Configuration
+﻿using Umbraco.Headless.Client.Net.Collections;
+using Umbraco.Headless.Client.Net.Delivery.Models;
+
+namespace Umbraco.Headless.Client.Net.Configuration
 {
     public interface IHeadlessConfiguration
     {
         string ProjectAlias { get; }
+        ITypeList<IContent> ContentModelTypes { get; }
```

Comment:
This would be a breaking change, so maybe consider adding another interface to add the `ITypeList<IContent> ContentModelTypes` and leave `IHeadlessConfiguration` untouched.
For the implementation and the public class we could then choose to only expose the one you implemented in `HeadlessConfiguration`.

We'd of course need to check on ContentModelTypes, so we can pass an empty list, but think that would be okay.

Reply:
oh, and since ContentDelivery is internal we should be able to change the constructor so we can change the interface that is passed in.